tests/unit/modules/test_win_file.py

```diff
@@ -57,8 +58,259 @@ def test_issue_43328_check_perms_no_ret(self):
             self.assertRaises(
                 CommandExecutionError, win_file.check_perms, self.FAKE_PATH)
 
-    @destructiveTest
-    @skipIf(not salt.utils.platform.is_windows(), 'Skip on Non-Windows systems')
+
+@destructiveTest
+@skipIf(NO_MOCK, NO_MOCK_REASON)
+@skipIf(not salt.utils.platform.is_windows(), 'Requires Pywin32 libraries')
+class WinFileCheckPermsTestCase(TestCase, LoaderModuleMockMixin):
+    '''
+    Test cases for the check_perms function in salt.modules.win_file
+    '''
+    temp_file = ''
+    current_user = ''
+
+    def setup_loader_modules(self):
+        self.current_user = salt.utils.win_functions.get_current_user(False)
+        return {
+            win_file: {
+                '__opts__': {
+                    'test': False}}}
+
+    def setUp(self):
+        self.temp_file = tempfile.NamedTemporaryFile(delete=False)
+        self.temp_file.close()
+        salt.utils.win_dacl.set_owner(obj_name=self.temp_file.name,
+                                      principal=self.current_user)
+        salt.utils.win_dacl.set_inheritance(obj_name=self.temp_file.name,
+                                            enabled=True)
+        self.assertEqual(
+            salt.utils.win_dacl.get_owner(obj_name=self.temp_file.name),
+            self.current_user)
+
+    def tearDown(self):
+        os.remove(self.temp_file.name)
+
+    def test_check_perms_set_owner_test_true(self):
+        '''
+        Test setting the owner of a file with test=True
+        '''
+        with patch.dict(win_file.__opts__, {'test': True}):
+            expected = {'comment': '',
+                        'changes': {},
+                        'pchanges': {'owner': 'Administrators'},
```

Comment:
Isn't pchanges going away? Does it still exist on develop?

Reply:
Good catch.  These were new tests added to 2019.2.  I'll update `pchanges` to be be `changes`.